Fastfile.private

```diff
@@ -155,42 +157,65 @@ reflected in `Info.plist` during building."
   desc "It uses the same password stored in keychain for the current user."
   desc "Make sure to store safely the output of this command in the right `Google Drive` folder."
   private_lane :generate_push_certificates do |options|
-    build_configuration = options[:build_configuration]
+
+    build_configuration_key = options[:build_configuration]
+    build_configuration = get_build_configuration(build_configuration: build_configuration_key)
+    match_type = get_match_type(build_configuration: build_configuration_key)
+    is_development = match_type == Actions::GetMatchTypeAction::MATCH_TYPES[:test]
+    output_path = "#{build_configuration_key.to_s}_push_certificates"
 
     desc "Bundle identifier from xCode project"
     bundle_identifier = read_project_property(
-      build_configuration: get_build_configuration(build_configuration: build_configuration),
+      build_configuration: build_configuration,
       build_setting: 'PRODUCT_BUNDLE_IDENTIFIER'
     )
 
+    UI.message "Attempting to create push certificates for build configuration '#{build_configuration}'"
+    UI.message "Creating push certificates as: '#{is_development ? 'development' : 'production'}' and bundle ID: '#{bundle_identifier}'"
```

Comment:
Instead of 'development' and 'production' shouldn't we use the build_configuration_key for those enums?
Or should we directly use the build configuration, since it can be production or release, right?

Reply:
This `development` and `production` refer to the certificate kind, not the configuration or scheme. For the sake of being explicit: 
<img width="234" alt="screen shot 2017-11-22 at 5 57 35 pm" src="https://user-images.githubusercontent.com/9341224/33149482-ab9dd4a2-cfae-11e7-8a78-080b0953ff23.png">

Reply:
👍